org.mozilla.Thunderbird.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "org.mozilla.Thunderbird",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "thunderbird",
+    "rename-icon": "thunderbird",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* OpenGL access */
+        "--device=dri",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Access to thunderbird configuration and cache files */
+        "--filesystem=~/.cache/thunderbird", "--filesystem=~/.thunderbird",
```

Comment:
Is it not possible to fix thunderbird to respect XDG (surely there are some upstream bugs or patches for this? or at the very least env vars flags to override it?)

Reply:
I don't know honestly, I've forked someone's abandoned project.

Reply:
[there's this 6 year old bug report about it](https://bugzilla.mozilla.org/show_bug.cgi?id=735285)

Reply:
@AdrianKoshka hah! I work with him ;-) I have my own [semi-abandoned fork](https://github.com/nedrichards/thunderbird-flatpak) of his repo as well.

Reply:
I'm personally not sure either about this now, but last time I checked I could not find a way to do that

Reply:
What about something like `"--persist=.cache/thunderbird", "--persist=.thunderbird"` instead?

Reply:
What's the difference between `--persist` and `--filesystem`?

Reply:
Persist is a good solution, there is an xdg cache dir but if they don't respect it then it doesn't matter.